qemu/tests/qemu_guest_agent.py

```diff
@@ -4274,6 +4277,40 @@ def gagent_check_run_qga_as_program(self, test, params, env):
         self.gagent_start(session, self.vm)
         time.sleep(5)
 
+    @error_context.context_aware
+    def gagent_check_driver_update_via_installer_tool(self, test, params, env):
+        """
+        Acceptance installer test:
+
+        1) Remove existed qga.
+        2) Install driver via virtio-win-guest-tools.exe.
+        3) Run virtio-win-guest-tools.exe signature check command in guest.
+        4) Verify the qemu-ga function works well.
+
+        :param test: QEMU test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment
+        """
+
+        run_install_cmd = params["run_install_cmd"]
+        installer_pkg_check_cmd = params["installer_pkg_check_cmd"]
+        gagent_uninstall_cmd = params["gagent_uninstall_cmd"]
+
+        vm = env.get_vm(params["main_vm"])
+        session = vm.wait_for_login()
+
+        uninstall_gagent(session, test, gagent_uninstall_cmd)
+        session = vm.reboot(session)
+        install_test_with_screen_on_desktop(vm, session, test, run_install_cmd,
+                                            installer_pkg_check_cmd,
+                                            copy_files_params=params)
+        win_installer_test(session, test, params)
```

Comment:
Hi @6-dehan We will finally remove gagent related steps on those installer cases, so here we can have the test like your current implementation, but do we need this `win_installer_test` in gagent test?

Reply:
oh, after checking on that function further, I think it's better to remove it, it's not related to gagent.

Reply:
yeah, I think we need  

> @6-dehan Is this ready for review?

nope, I don't add the content that we talked on our last meeting. I'll work on it asap.